DM-7803: add time series launch capability to suit app.

loitly opened the pull request:
https://jira.lsstcorp.org/browse/DM-7803

On LSST PDAC, when searching for 'Deep Forced Source' data, there should be a button to view the time series data for the highlighted objectId.
If you upload a table with time series's metadata, there should be a button to view the table as time series.

To Test:
- Check out 'DM-7803_triview_timeseries' branch in both firefly and suit repo.
- Build suit and then goto http://localhost:8080/suit/
- Search for "Deep Forced Source", then click on 'View Time Series' button at upper right corner to view the data in a separate window.
- When uploading a table with time series metadata, i.e. 'timeCName', there should also be a 'View as Time Series' button at the same location

robyww commented on src/firefly/js/tables/ui/ActiveTableButton.jsx:
```diff
@@ -0,0 +1,73 @@
+ *    or a message prompting the user to select rows first
+ */
+export class ActiveTableButton extends Component {
+
```
I had a little trouble figuring out how this all worked.  The key is the onChanges function that takes a table_id and a highlighted row and return a boolean and a title.  I think this should be documented.

loitly commented:
Changes since last pushed:
- fix highlighted row out of sync after sorting or filtering.
- for lsst, show only first of multiimage. - trey
- add bolded text to highlight the time series button.

tgoldina commented on src/firefly/js/templates/lightcurve/LcManager.js:
```diff
@@ -321,13 +328,35 @@ function handleAddChart(layoutInfo, chartId) {
+    const tbl_id = get(action, 'payload.request.META_INFO.tbl_id');
+    if (tbl_id === LC.RAW_TABLE) {
+        removeTablesFromGroup();
+        removeTablesFromGroup(LC.PERIODOGRAM_GROUP);
```
Looks like now we clear the derived tables only on the new search. But if we don't remove the derived tables, when raw table changes, how do we keep raw table in sync with the related tables? If user filters Raw and Phase Folded tables separately, we can not synchronize the highlight index between them, can we?

loitly replied:
> tgoldina: ... how do we keep raw table in sync with the related tables? If user filters Raw and Phase Folded tables separately, we can not synchronize the highlight index between them, can we?

You are right.  I think a filter of a raw table should trigger a reset condition.

hello
